roles/common/files/uug_ansible_wrapper.py

```diff
@@ -578,12 +574,106 @@ def get_distro_release_name():
+                    '1': DEFAULT_GIT_REMOTE
+                }
+            )
+            header = "using testing branch master"
+        elif branch_mismatch and looks_minty and system_exists and chosen_exists:
+            warning_prompt = (
+                "You are using a version of the configuration tool meant for a different release of Linux Mint. " 
+                "You should consider upgrading branches. "
+                "The recommended configuration settings for your release version of Linux Mint are:"
+                "\nRelease: %(0)s and URL: %(1)s" % {
+                    '0': system_version,
+                    '1': DEFAULT_GIT_REMOTE
+                }
+            )
+            header = "Current branch is outdated"
+            warning_bool = False
+        elif branch_mismatch and looks_minty and system_exists and (not chosen_exists):
+            warning_prompt = (
+                "Your current Linux Mint version does not exist as a branch on the specified remote url. "
+                "You should consider switching to the master branch. "
+                "The recommended configuration settings for your release version of Linux Mint are:" 
+                "\nRelease: %(0)s and URL: %(1)s" % {
+                    '0': system_version,
+                    '1': DEFAULT_GIT_REMOTE
+                }
+            )
+            header = "Linux Mint version not found on URL"
+            warning_bool = False
+        elif system_exists and (not chosen_exists):
+            warning_prompt = (
+                "Your currently chosen branch does not exist on the chosen url. "
+                "You should fix your remote url or switch to a branch that exist on that url. "
+                "The recommended configuration settings for your release version of Linux Mint are:" 
+                "\nRelease: %(0)s and URL: %(1)s" % {
+                    '0': system_version,
+                    '1': DEFAULT_GIT_REMOTE
+                }
+            )
+            header = "Branch does not exist on specified remote URL"
+            warning_bool = False
+        elif (not system_exists) and (not chosen_exists):
+            warning_prompt = (
+                "We do not support your current OS and the branch chosen does not exist on the chosen remote URL. "
+                "You should consider switching to the master branch. "
+                "The recommended configuration settings for your release version of Linux Mint are:" 
+                "\nRelease: %(0)s and URL: %(1)s" % {
+                    '0': "master",
+                    '1': DEFAULT_GIT_REMOTE
+                }
+            )
+            header = "OS not supported and branch not found"
+            warning_bool = False
+        
+    if header and warning_prompt:
+        show_dialog(
+            self, Gtk.MessageType.WARNING, Gtk.ButtonsType.OK, header, warning_prompt
+        )
+            
+    return warning_bool
```

Comment:
```suggestion
    return warning_prompt is None
```

Reply:
If `warning_prompt is not None` then there was a warning set and so it should have the same value as `warning_bool`.